spacy_curated_transformers/tests/test_cli_app.py

```diff
@@ -1,9 +1,130 @@
+    name = "explosion-testing/xlm-roberta-test"
+    """,
+    """
+    [nlp]
+    lang = "en"
+    pipeline = ["transformer"]
+    [components]
+    [components.transformer]
+    factory = "curated_transformer"
+    [components.transformer.model]
+    @architectures = "spacy-curated-transformers.AlbertTransformer.v1"
+    [initialize]
+    [initialize.components]
+    [initialize.components.transformer]
+    [initialize.components.transformer.encoder_loader]
+    @model_loaders = "spacy-curated-transformers.HFTransformerEncoderLoader.v1"
+    name = "explosion-testing/albert-test"
+    """,
+    """
+    [nlp]
+    lang = "en"
+    pipeline = ["transformer"]
+    [components]
+    [components.transformer]
+    factory = "curated_transformer"
+    [components.transformer.model]
+    @architectures = "spacy-curated-transformers.BertTransformer.v1"
+    [initialize]
+    [initialize.components]
+    [initialize.components.transformer]
+    [initialize.components.transformer.encoder_loader]
+    @model_loaders = "spacy-curated-transformers.HFTransformerEncoderLoader.v1"
+    name = "explosion-testing/bert-test"
+    """,
+    """
+    [nlp]
+    lang = "en"
+    pipeline = ["transformer"]
+    [components]
+    [components.transformer]
+    factory = "curated_transformer"
+    [components.transformer.model]
+    @architectures = "spacy-curated-transformers.CamembertTransformer.v1"
+    [initialize]
+    [initialize.components]
+    [initialize.components.transformer]
+    [initialize.components.transformer.encoder_loader]
+    @model_loaders = "spacy-curated-transformers.HFTransformerEncoderLoader.v1"
+    name = "explosion-testing/camembert-test"
+    """,
+    """
+    [nlp]
+    lang = "en"
+    pipeline = ["transformer"]
+    [components]
+    [components.transformer]
+    factory = "curated_transformer"
+    [components.transformer.model]
+    @architectures = "spacy-curated-transformers.RobertaTransformer.v1"
+    [initialize]
+    [initialize.components]
+    [initialize.components.transformer]
+    [initialize.components.transformer.encoder_loader]
+    @model_loaders = "spacy-curated-transformers.HFTransformerEncoderLoader.v1"
+    name = "explosion-testing/roberta-test"
+    """,
+]
+
+
+@pytest.mark.slow
+@pytest.mark.skipif(not has_huggingface_hub, reason="requires Hugging Face Hub")
+@pytest.mark.skipif(
+    not has_hf_transformers, reason="requires Hugging Face transformers"
+)
+@pytest.mark.parametrize("config", FILL_TRANSFORMER_CONFIG_STRS)
+def test_fill_config_transformer(config):
+    with make_tempdir() as d:
+        file_path = d / "test_conf"
+        with open(file_path, "w", encoding="utf8") as f:
+            f.writelines([config])
+
+        result = CliRunner().invoke(
+            app,
+            [
+                "init",
+                "fill-config-transformer",
+                str(file_path),
+                "-",
+            ],
+        )
+        try:
+            assert result.exit_code == 0
```

Comment:
Can we add checks for some of the expected values for the models above?

(Is the result `model_max_length = 9999` right for `explosion-testing/roberta-test`?)

Reply:
Added checks for the expected values.